src/_reducers/HelloWorldReducer.js

```diff
@@ -0,0 +1,15 @@
+import StateStorage from '../_store/StateStorage';
+
+const SayHello = 'Hello';
+const SayHi = 'Hi';
+const NoSayHello = '';
+
+const initialState = new Map({subject: 'World'});
```

Comment:
You need to import 'immutable' or else you are using the ES6 Map. Which will lead to unexpected behavior ;) One solution is to always use Immutable.Map, another is to continue being cool and using it like this but being mindful. ;)